src/java/test/org/apache/zookeeper/test/ClientPortBindTest.java

```diff
@@ -102,9 +102,9 @@ public void testBindByAddress() throws Exception {
         startSignal = new CountDownLatch(1);
         ZooKeeper zk = new ZooKeeper(HOSTPORT, CONNECTION_TIMEOUT, this);
         try {
-            startSignal.await(CONNECTION_TIMEOUT,
```

Comment:
The change in this file sounds not related to this patch. Usually we tend to limit changes only relevant to a patch. Similar for the change in ReadOnlyModeTest file. It's fine in this case as both changes are small and are good improvements.